src/main/java/com/amazonaws/services/dynamodbv2/datamodeling/internal/Base64.java

```diff
@@ -0,0 +1,35 @@
+package com.amazonaws.services.dynamodbv2.datamodeling.internal;
+
+import java.util.Base64.Decoder;
+import java.util.Base64.Encoder;
+
+/**
+ * A class for decoding Base64 strings and encoding bytes as Base64 strings.
+ */
+public class Base64 {
+    private static final Decoder DECODER = java.util.Base64.getMimeDecoder();
```

Comment:
Why `getMimeDecoder` rather than `getDecoder`?

Reply:
From the Readme:

```
To use this SDK you must have:

    A Java 8 development environment
```

Mime decoder ignores illegal characters, which is needed because SDK1 leaves the choice of encoder up to the jaxb serializer.

Reply:
> A Java 8 development environment
>
> If you do not have one, go to Java SE Downloads on the Oracle website, then download and install the Java SE Development Kit (JDK). *Java 8 or higher is recommended.*

The messaging is currently confused. I think just changing "recommended" for "required" should be fine.

Reply:
Hmm, agreed. This blurb is reused between the encryption sdk and here, so I'll update both to say required instead.

Reply:
Updated README to state Java 8 is required. Another one is here, for the encryption sdk for java: https://github.com/aws/aws-encryption-sdk-java/pull/98